packages/core/types/src/types/core/schemas/index.ts

```diff
@@ -47,6 +47,11 @@ export interface Schema {
    * Custom table name for the schema
    */
   collectionName?: string;
+
+  /**
+   * Information about schema naming and display
+   */
+  info: Info;
```

Comment:
Should this be optional?

```suggestion
  info?: Info;
```